TESTING.md

```diff
@@ -249,6 +249,39 @@ cargo make watch-test
 
 **Note:** For more information on publishing AMIs see [publishing](PUBLISHING.md).
 
+### vmware-k8s
+You need to [build](BUILDING.md) Bottlerocket and a TUF repo to test VMware variant.
```

Comment:
Need a line break.

```suggestion

You need to [build](BUILDING.md) Bottlerocket and a publicly accessible TUF repoistory to test VMware variants.
```
Can also hyperlink "TUF repo" to https://github.com/bottlerocket-os/bottlerocket/blob/develop/PUBLISHING.md#repo-location